server/src/test/java/keywhiz/service/resources/automation/v2/SecretResourceTestHelper.java

```diff
@@ -35,10 +39,10 @@ public class SecretResourceTestHelper {
     this.mapper = mapper;
   }
 
-  Response create(CreateSecretRequestV2 request) throws IOException {
-    RequestBody body = RequestBody.create(JSON, mapper.writeValueAsString(request));
+  Response create(CreateSecretRequestV2 request) {
+    RequestBody body = RequestBody.create(JSON, toJson(request));
     Request post = clientRequest("/automation/v2/secrets").post(body).build();
-    return mutualSslClient.newCall(post).execute();
+    return execute(post);
   }
```

Comment:
Constantly being nagged about exception handling while writing tests drives me nuts, so I modified this and several other test helper methods in this class to use unchecked exceptions.